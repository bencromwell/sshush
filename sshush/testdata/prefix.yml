---
global:
  UseRoaming: "no"

default:
  User: ben
  IdentityFile: ~/.ssh/id_rsa

web_servers:
  Prefix: projects-
  Config:
    Port: 2201
    IdentityFile: ~/.ssh/digital_ocean
  Hosts:
    do-1:
      HostName: projects-do-1.example.com
    do-2:
      HostName: projects-do-2.example.com
    aws:
      HostName: projects-aws.example.com
      IdentityFile: ~/.ssh/aws
