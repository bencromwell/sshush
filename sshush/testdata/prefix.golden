# Generated by sshush v0.0.0-dev
# From testdata/prefix.yml

# web_servers
Host projects-aws
    HostName projects-aws.example.com
    IdentityFile ~/.ssh/aws
    Port 2201
    User ben

Host projects-do-1
    HostName projects-do-1.example.com
    IdentityFile ~/.ssh/digital_ocean
    Port 2201
    User ben

Host projects-do-2
    HostName projects-do-2.example.com
    IdentityFile ~/.ssh/digital_ocean
    Port 2201
    User ben

# Global config
Host *
    UseRoaming no
